pages/common/xz.md

```diff
@@ -0,0 +1,23 @@
+# xz
+
+> Compress or decompress .xz and .lzma files.
+
+- Compress a file:
+
+`xz {{file}}`
+
+- Decompress a file:
+
+`xz -d {{file.xz}}`
+
+- Decompress a file and write to stdout:
+
+`xz -dc {{file.xz}}`
+
+- Compress a file, but don't delete the original:
+
+`xz -k {{file}}`
+
+- Compress a file and attempt to improve compression ratio by using more CPU time:
```

Comment:
I tend to sort of copy the wording the original man page uses. :stuck_out_tongue: Actually, I'm realizing that the `-e` flag doesn't make sense without a `-0`...`-9` flag as well. I'm gonna drop this example and use those instead (I'm typically too lazy to use anything but default compression ratio, but I imagine anyone interested enough in compression to not just use the more-popular gzip might care about compression ratios).